tst/testinstall/stringobj.tst

```diff
@@ -119,10 +119,11 @@ gap> s := ReadCSV( filename{[1..Length(filename)-4]}, ";" );
 [ rec( f := 2, f1 := 1, f_2 := 3 ), rec( f1 := 4, f_2 := 6 ), rec( f1 := 7 ), 
   rec( f := 11, f_2 := 12 ), rec( f := "yy", f1 := "x", f_2 := "zzz" ) ]
 gap> tmpdir := DirectoryTemporary();;
-gap> fname := Filename(tmpdir, "output.csv");;
-gap> PrintCSV( fname, s );
-gap> s = ReadCSV( fname );
-true
+gap> for i in [1..1000] do
+>   fname := Filename(tmpdir, "output.csv");;
+>   PrintCSV( fname, s );
+>   if s <> ReadCSV( fname ) then Print("failed loop", i,"\n"); fi;
```

Comment:
If I look at this test in a year, I won't have a clue what it is about. I'd have to dig through the commit history. Perhaps a brief comment to explain that could be added?

Reply:
Good idea, now fixed